src/main/java/com/mraof/minestuck/item/AncientThumbDrive.java

```diff
@@ -0,0 +1,49 @@
+package com.mraof.minestuck.item;
+
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TranslatableComponent;
```

Comment:
remove unused import

Reply:
all of those are used

Reply:
Likely referring to the thing that no longer exists in 1.19.2.